pages/linux/bmon.md

```diff
@@ -0,0 +1,19 @@
+# bmon
+
+> Monitor bandwidth and capture network related statistics.
+
+- Display all interfaces:
+
+`bmon -a`
+
+- Display rates in bits per second:
+
+`bmon -b`
+
+- Set policy to define which network interfaces are displayed:
+
+`bmon -p {{interfaces}}`
```

Comment:
You can list multiple interfaces? Perhaps a concrete example would help here.

Reply:
Updated for clarity